packages/app/src/app/hooks/useWorkspaceAuthorization.ts

```diff
@@ -18,7 +18,7 @@ export const useWorkspaceAuthorization = () => {
    * Team states
    */
 
-  const isTeamSpace = !isPersonalSpace;
+  const isTeamSpace = activeTeam !== null && !isPersonalSpace;
```

Comment:
isTeamSpace should be true only after the team is loaded (unfortunately null means the team is not yet there in this case)